tests/test_utils/test_transactiontestcase.py

```diff
@@ -1,7 +1,27 @@
+
```

Comment:
It was there in 2d2758f66d4c4cd05ca7f5d576943db5dbd04f54 but I guess you updated it in the meantime and github somehow moved my comment.